examples/ultrahdr_app.cpp

```diff
@@ -298,6 +296,10 @@ class UltraHdrAppInput {
         mQuality(95),
         mOTf(oTf),
         mOfmt(oFmt),
+        mFullRange(UHDR_CR_UNSPECIFIED),
+        mMapDimensionScaleFactor(1),
```

Comment:
A minor comment: I think it's better to keep it 4 as default here, which is aligned with the original code and other places in this change.

Reply:
Done